src/quick-lint-js/fe/parse-class.cpp

```diff
@@ -1126,6 +1129,40 @@ void parser::parse_and_visit_class_or_interface_member(
       }
     }
 
+    void error_if_access_specifier_not_first_in_field() {
+      if (p->options_.typescript && modifiers.size()) {
```

Comment:
Nit: Check .empty rather than .size:
```suggestion
      if (p->options_.typescript && !modifiers.empty()) {
```